doc/EnvironmentalMonitoring/Monitoring-architecture.md

```diff
@@ -126,10 +128,24 @@ The following sensors produce monitoring data:
 
 Monitoring service is installed as a separate package, with name `xroad-monitor`. It runs in a separate process.
 
-The service also publishes the monitoring data via JMX. Local monitoring agents can use this as an alternative way to fetch monitoring data. With default configuration, JMX access is only allowed from localhost.
+#### 2.2.1 JMX interface
+
+The service also publishes the monitoring data via JMX. Local monitoring agents can use this as an alternative way to fetch monitoring data. With the default configuration, JMX is disabled.
 
 ![monitoring JMX agent](img/monitoring-jmx.png)
 
+JMX is enabled by adding the required configuration in `/etc/xroad/services/local.conf` file. The file is opened for editing and changes are made on the `PROXY_PARAMS` variable value. After the `PROXY_PARAMS` variable value has been updated, the `xroad-monitor` service must be restarted.
```

Comment:
`PROXY_PARAMS` should be `MONITOR_PARAMS`

Reply:
Copy/paste error...fixed.